modules/integration_azure-datafactory/conf/04-adf-integration-available-memory.yaml

```diff
@@ -0,0 +1,21 @@
+module: "Azure DataFactory"
+name: "Available Memory"
+filtering: "filter('resource_type', 'Microsoft.DataFactory/factories') and filter('primary_aggregation_type', 'true')"
+aggregation: ".sum(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
+value_unit: "MB"
+transformation: ".min(over='15m')"
+signals:
+  memory:
+    metric: "IntegrationRuntimeAvailableMemory"
+    rollup: average
+  signal:
+    formula:
+      memory.scale(0.000000953674316) # Scale to MB unit
+rules:
+  critical:
+    threshold: 512
```

Comment:
I think we need either a note in readme to explain why this value makes sens for a general usage or to remove it else to let the user customize the value suited to its need.